ale_linters/ocaml/merlin.vim

```diff
@@ -0,0 +1,20 @@
+" Author: Andrey Popp -- @andreypopp
+" Description: Report errors in OCaml code with Merlin
+
+if !exists('g:merlin')
+  echo 'Error: merlin vim bindings are required for Ale merlin linter to work'
```

Comment:
Am I right that you want me to keep `echo` statement but point users to ale's doc instead of the current message there?